src/main/resources/com/google/api/codegen/java/standalone_sample.snip

```diff
@@ -4,37 +4,17 @@
 #FIXME: currently the variable of output of an api call is hardcoded to "response"
 # and "response_item". We should fix it by saving the variable name in the view model
 @snippet generate(sampleFile)
-  //// [ This is an auto-generated sample file produced by the gapic-generator. Sample name: "{@sampleFile.classView.name}" ]
-  //// STUB standalone sample "{@sampleFile.classView.name}" /////
-
-  // FIXME: Insert here set-up comments that we never want to display in cloudsite. These are seen by users perusing the samples directly in the repository.
-
   @let apiMethod = sampleFile.classView.libraryMethod, \
        sample = apiMethod.samples.get(0)
-    // [START {@sample.regionTag}]
-
-    // FIXME: Insert here boilerplate code not directly related to the method call itself.
-
-    //      calling form: "{@sample.callingForm.toString}"
-    //        region tag: "{@sample.regionTag}"
-    //         className: "{@sampleFile.classView.name}"
-    //          valueSet: "{@sample.valueSet.id}" ("{@sample.valueSet.title}")
-    //       description: "{@sample.valueSet.description}"
-    //        {@sample.valueSet.parameters}
-    //      apiMethod "{@apiMethod.name}" of type "{@apiMethod.type}"
-
-    // FIXME: Insert here code to prepare the request fields, make the call, process the response.
-    # FIXME: change the package name from pubsub to library
-
-    package {@sampleFile.fileHeader.packageName};
+    package {@sampleFile.fileHeader.examplePackageName};
```

Comment:
However, we should keep at the top of the file the line clarifying this is an autogenerated file:
```
// DO NOT EDIT! This is a generated sample ("Request",  "pi_version")
```

We do this in Python and PHP.

Reply:
whoops :/ too happy deleting things :)